mindocr/optim/adamw.py

```diff
@@ -164,6 +165,7 @@ def __init__(
         self.reciprocal_scale = Tensor(1.0 / loss_scale, ms.float32)
         self.clip = clip
 
+    @ms_function
```

Comment:
`@ms_function` is deprecated in MS2.0 and higher.

Reply:
ok

Reply:
But in MS1.9 there's no `jit`. Can do this way:
```python
from utils.misc import is_ms_version_2

if is_ms_version_2():
    from mindspore import jit
else:
    from mindspore import ms_function as jit
```
https://github.com/mindspore-lab/mindocr/blob/1249bc0aea91112dd641b1767eabe458e3452173/mindocr/utils/misc.py#L69